digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/TierekisteriTestApi.scala

```diff
@@ -237,6 +315,14 @@ class TierekisteriTestApi extends ScalatraServlet with JacksonJsonSupport {
       roadWidth
     } else if (fieldCode == trafficSignTRCode) {
       trafficSign
+    } else if (fieldCode == laneArragementTRCode) {
+      massTransitLane
+    } else if (fieldCode == pavedRoadTRCode) {
+      pavedRoad
+    } else if (fieldCode == damagedByThawTRCode) {
+      damagedByThaw
+    } else if (fieldCode == europeanRoadTRCode) {
+      europeanRoad
```

Comment:
I think is not necessary, because it's like Faustino says, on next time, we don't add cases on this TesApi because nobody use it anymore. It's work for nothing.